build.config.ts

```diff
@@ -8,7 +8,14 @@ export default defineBuildConfig({
       exportConditions: ['production', 'node'] as any,
     },
   },
-  entries: ['src/index'],
+  entries: [
+    'src/index', 
+    {
+      builder: 'mkdist',
```

Comment:
Since utils should be tree-shakable it would be the best if we use a bundled entity. It also assure we can reuse same code and keep bundle size minimal